### ISSUE TITLE
Able to make infinite obsidian in the nether bug

### ISSUE THREAD

**Issue by Flying--Dutchman** [cppchriscpp/TravelPortals#28]
Like issue #10 , you can create infinite obsidian in the nether.

Steps to reproduce:
1. Build portal
2. Remove blocks under the portal (where the water is)
3. Remove the door

**Comment by Phoenix616** [comment]
Do you mean that the water flows from the portal? Because that should be blocked by [this listener](https://github.com/cppchriscpp/TravelPortals/blob/master/src/main/java/net/cpprograms/minecraft/TravelPortals/TravelPortalsBlockListener.java#L168-L183) already. If you mean a different setup then please provide some more info about this.

Also could you please provide your used server and plugin version please? Maybe something changed about that in newer server versions.

**Comment by Flying--Dutchman** [comment]
Yes the water flows from the portal, but only if the steps are followed.

Build the portal:
![1](https://user-images.githubusercontent.com/9158539/93689708-736f3c00-fad1-11ea-9ba3-87e1a3a0c215.png)

Remove the block under the water (second yellow block)
![2](https://user-images.githubusercontent.com/9158539/93689721-88e46600-fad1-11ea-8664-e9c86f096b30.png)
-->
![3](https://user-images.githubusercontent.com/9158539/93689724-90a40a80-fad1-11ea-85d0-6ea307d0e0b7.png)

Remove the block under the door, and you get this:
![4](https://user-images.githubusercontent.com/9158539/93689733-9c8fcc80-fad1-11ea-98b2-4d78183d1c47.png)


Running on 1.16.2, plugin build from source of master branch.

**Comment by Phoenix616** [comment]
I think I see why this is an issue. I assume the portal itself stays? Can you confirm that with `/portal info` or the lsit? Because it might not recognise that the portal was broken due to the block below the door getting broken and not the door itself. And because of that doesn't remove the water blocks.

**Comment by Flying--Dutchman** [comment]
Maybe as a quick "fix" would be to always use lava in the nether and listen on lava damage.
If someone than breaks the portal like described, there is only a lot more lava.

Real Nether portals, or even end portals, would be awesome, but I could guess that that would interfere with other plugins?
But then again, Advanced Portals also seems to use the nether portal.... 

But yeah, should definitly be an option
